uint8 status
Header header
float64[49] Alinear
float64[14] Blinear
float64[7] Clinear